src/main/java/me/arminb/sara/controller/UserController.java

```diff
@@ -0,0 +1,73 @@
+package me.arminb.sara.controller;
+
+import me.arminb.sara.entities.User;
+import me.arminb.sara.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<List<User>> findAll() {
+        try {
+            List<User> users = userService.findAll();
+            return new ResponseEntity<List<User>>(users, HttpStatus.OK);
+        }
+        catch (Exception e){
```

Comment:
Catching Exception is the worst thing to do in this layer or anywhere. You should be specific about the type of exceptions that need to be catch and do appropriate actions corresponding to that specific exception. 

In this layer, you should translate the return values and exceptions received from the lower layers to appropriate error codes. For example, if the null return value means the request object is not found, you should return 404 (Not Found) status code. Another example would be returning 500 (Server Error) status code when receiving DataAccessException.

Later we will discuss how to handle error codes in a better way because the number of HTTP error codes is limited and in a more complex situation is not sufficient for covering all of the cases.